src/main/webapp/searchcharities.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>User Page</title>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="/css/main.css">
+    <link rel="stylesheet" href="/css/user-page.css">
+    <script src="/js/search-charities.js"></script>
+</head>
+<body>
+<nav>
+    <ul id="navigation">
+        <li><a href="/">Home</a></li>
+        <li><a href="/aboutus.html">About Our Team</a></li>
+    </ul>
+</nav>
+<h1 id="page-title">Charities Search Page</h1>
```

Comment:
to show this the right way, you can use :
<Label id="page-title" class="h1-label">Charities Search Page</Label>

Reply:
It does not show the full text because of "<", ">", so I texted you on the chat